xml/ldap_searchresult.xml

```diff
@@ -657,7 +657,7 @@
     <param pos="0" name="service.product" value="UnboundID Directory Proxy Server"/>
     <param pos="1" name="service.version"/>
   </fingerprint>
-  <fingerprint pattern="(?im:namingContexts1.\x04.cn=.?pbx.*\x04.ldapServiceName1.\x04.IPVA-\w+-)">
+  <fingerprint pattern="(?im:namingContexts1.\x04.cn=.?pbx.*\x04.ldapServiceName1.\x04.IPVA-\w+-)" flags="REG_MULTILINE">
```

Comment:
FYI, we haven't acted on this yet, but we have desires to remove support for `flags` entirely, but Go support and this specific newline problem is one of the complications we identified.  That shouldn't change what you are doing in this PR just yet, but just FYI.

Is another option to explicitly list the carriage return and/or newline characters that might separate parts of this rather than `.*`?

Reply:
Specifying the newline in the match would work as well, but it might take a while to fix the existing fingerprints. Let me know if I can help either way.

Reply:
Regarding newlines and matching, the Go syntax.Regexp documentation (page)[https://golang.org/src/regexp/syntax/parse.go?s=1700:1717#L43] goes into the distinction here. For the implementation in recog-go, I use REG_MULTILINE to set MatchNL (DotNL | ClassNL). The latter is needed for things like \s to match newlines.

Reply:
Anther option might be to assume REG_MULTILINE is on for _all_ patterns. This seemed to break a small number of matches when I tested earlier, due to .* including whitespace, but I think the total was under 5. The REG_ICASE stuff seems to work just fine via (?i:pattern) prefixes, but (?m) gets wonky without `.` and `\s` matching newline.